src/routes/pages/en/zaps.md

```diff
@@ -22,12 +22,12 @@ From the beginning of the Nostr protocol, it was common to see Lightning invoice
 We won't get into the deep technical weeds here but for the curious among you, let's look at the basic mechanics of how Zaps work.
 
 1. When you click or tap on the little ⚡ icon in your client (Damus, Iris, Amethyst, etc), the first thing that happens is that the client pings the [LNURL server](https://thebitcoinmanual.com/articles/what-is-ln-url-and-how-does-it-work/) that sits in front of lightning wallet of the person that you're trying to Zap. The first request goes something like, "Hi there, I would love to give Alice some sats."
-1. The LNURL server responds and, if Alice's wallet supports Zaps, it will tell the client so and send/confirm Alice's public key.
-1. At this point, the client does a little work to put together a Zap request (a kind 9734 note) with data about the profile or note that it would like to Zap, the amount, the relays it should broadcast the note to, and a few other things. This is effectively a request for an invoice from the LNURL server.
-1. The LNURL server responds with the requested invoice.
-1. At this point the client will hand that invoice off to the user's lightning wallet to be paid. If you are using a wallet like Alby in the browser (and have set a budget) this process can happen very quickly.
-1. Once the user has paid the invoice directly to the wallet of the person they're zapping, the receiver's wallet will create a kind 9735 note and then broadcast that to the relays specified earlier in the Zap request.
-1. Relays receiving this note will then be able to tell connected clients about the zap and clients and show the zap to users in their UI.
+2. The LNURL server responds and, if Alice's wallet supports Zaps, it will tell the client so and send/confirm Alice's public key.
```

Comment:
@Flobstr no big deal but having just `1.` at the beginning of each line is valid Markdown and will be translated into a properly numbered list. I usually default to using only `1.` at the beginning so that I don't have to go back and renumber things if I later want to add an item to the list.

Reply:
Good to know and makes totally sense. I'm leaning a lot right now